### PR TITLE
add standard src scripts

### PR DESCRIPTION
**Pull request by IsaackMwenda**
This adds standard src scripts copied from IOM with;
1. An update to age categorization in code imputation to match current values in covid19-ke
2. Addition of analysis util
3. Update  Dockerfile to add plotly dependencies as is on covid19-ke

Will update the scripts location e.g code imputation in the next pr

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+tMIT License
```
Comment (as2388) [inline LICENSE]: typo

File: Pipfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+name = "pypi"
+
+[packages]
+CoreDataModules = {editable = true,git = "https://www.github.com/AfricasVoices/CoreDataModules",ref = "v0.14.0"}
+PipelineInfrastructure = {editable = true,git = "https://www.github.com/AfricasVoices/Pipeline-Infrastructure",ref = "v0.0.4"}
+RapidProTools = {editable = true,git = "https://www.github.com/AfricasVoices/RapidProTools",ref = "v0.3.3"}
+pytz = "*"
+python-dateutil = ">=2.8.0"  # Need this minimum so that isoparse handles T24:00:00 correctly.
+google-cloud-storage = "*"
+altair = "*"
```
Comment (as2388) [inline Pipfile]: Plotly

File: generate_analysis_graphs.py [hunk 3]
```diff
@@ -0,0 +1,159 @@
+import argparse
+import glob
+import json
+from collections import OrderedDict
+
+import altair
```
Comment (as2388) [inline generate_analysis_graphs.py]: plotly

**Review by as2388 — APPROVED** [review]
Skimmed most of it because copy/paste.

Re the switch from altair to plotly - you'll need to update the pipfile, lockfile, and generate_analysis_graphs if you update the Dockerfile. I think it would be better to revert the Dockerfile to altair, get the pipeline working up to push to coda (which is the most important part at the moment), with the basic automated analysis in altair, then upgrade to plotly later.

File: generate_analysis_graphs.py (hunk 3 above)
Comment (IsaackMwenda) [reply generate_analysis_graphs.py]: updated Thanks.

File: Pipfile (hunk 2 above)
Comment (IsaackMwenda) [reply Pipfile]: updated Thanks.

File: LICENSE (hunk 1 above)
Comment (IsaackMwenda) [reply LICENSE]: updated Thanks.

**Pull request merged**